### ISSUE TITLE
this.config.template can t be absolute path

### ISSUE THREAD

**Issue by maboiteaspam** [1000ch/node-github-markdown#4]
Hi,

in index.js, the template property of the config object can t be passed as an absolute path.

It is always resolved to module path + path template.

I d like to suggest to change this line

```js
 this.template = this.config.template ? path.join(process.cwd(), this.config.template) : __dirname + '/assets/template.jade';
```

to 

```js
 this.template = this.config.template || __dirname + '/assets/template.jade';
```

Or anything else you consider as more suitable.

Tanks !

**Comment by 1000ch** [comment]
@maboiteaspam Thank you for suggestion.
This problem may be fixed at version `0.4.3`.
Please update and try agan? :)

**Comment by maboiteaspam** [comment]
cool! That s great, thanks a lot : )

If it can be any interest to you, i used it here, https://github.com/maboiteaspam/readit
